leaderboardsgg/leaderboard-site
# [LBGG-533] Account Login & Creation user feedback

## What

Short description of what was added/changed/removed in this PR

## Link to Issue

Closes https://github.com/leaderboardsgg/leaderboard-site/issues/563

## Acceptance

### Steps for testing

- Ensure jest tests pass
- Ensure Login and account creation shows error messages after backend requests

### Images

![image](https://github.com/leaderboardsgg/leaderboard-site/assets/11199709/f477562f-3320-4538-a080-64138db855f4)
![image](https://github.com/leaderboardsgg/leaderboard-site/assets/11199709/13844867-84fa-45c1-ae2c-519131868923)
